From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: William Blake Galbreath <blake.galbreath@gmail.com>
Date: Wed, 5 Jul 2023 12:48:08 -0500
Subject: [PATCH] moded fix


diff --git a/src/main/java/org/bukkit/plugin/messaging/StandardMessenger.java b/src/main/java/org/bukkit/plugin/messaging/StandardMessenger.java
index 1092cb46bbe1bcfc46e1630382fbf066099c32fc..2b1f0405614c8aeb98bdd3bc80c4e734bf96717e 100644
--- a/src/main/java/org/bukkit/plugin/messaging/StandardMessenger.java
+++ b/src/main/java/org/bukkit/plugin/messaging/StandardMessenger.java
@@ -496,7 +496,7 @@
         if (channel.equals("bungeecord:main")) {
             return "BungeeCord";
         }
-        if (channel.length() > Messenger.MAX_CHANNEL_SIZE) {
+        if (channel.length() > 256) {
             throw new ChannelNameTooLongException(channel);
         }
         if (channel.indexOf(':') == -1) {
